<launch>
	
	<param name="robot_description" textfile="$(find ros_learning)/urdf/scara.urdf" />
	<node pkg="ros_learning" name="test" type="test_ik.py" output="screen"/>
	<node name="robot_state_publisher" pkg="robot_state_publisher" type="robot_state_publisher" />
    <node name="rviz" pkg="rviz" type="rviz" />
    	
</launch>